src/timedate/timedatectl.c

```diff
@@ -261,6 +261,15 @@ static int set_local_rtc(int argc, char **argv, void *userdata) {
         if (b < 0)
                 return log_error_errno(b, "Failed to parse local RTC setting '%s': %m", argv[1]);
 
+        if (b == 1)
+		printf("\n%s"
```

Comment:
log_warning, rather than printf

Reply:
true. in fact the other location this message is printed, should also be switch to use log_warning() (with an fflush(stdout); before)

Reply:
Something like this should be fine I think
 ```
 if (b == 1) {
                fflush(stdout);
                log_warning("Warning: The system is now being configured to read the RTC time in the local time zone. This mode cannot be fully supported. It will create various problems with time zone changes and DST adjustments. If at all possible, use RTC in UTC by setting 'set-local-rtc' to \'0\' instead of \'1\'");
        }
```